javascript/builtins/Intl.json

```diff
@@ -975,7 +975,7 @@
                 "version_added": true
               },
               "webview_android": {
-                "version_added": "4.4"
+                "version_added": "37"
```

Comment:
Okay, I see, 4.4 should actually be 37. That's where the other 37 versions come from as well? Can you explain why this is?